src/dlgPackageExporter.h

```diff
@@ -86,20 +86,21 @@ private slots:
     void slot_openPackageLocation();
     void slot_packageChanged(int);
     void slot_updateLocationPlaceholder();
-    void slot_enableExportButton(const QString &text);
+    void slot_enableExportButton(const QString& text);
```

Comment:
Actually - this is no longer being used as a `SLOT` method so doesn't want/need to be here AFAICT.

Reply:
It is still: `connect(ui->lineEdit_packageName, &QLineEdit::textChanged, this, &dlgPackageExporter::slot_enableExportButton);`

Reply:
Strange - I dunno why my cross-project search for that didn't find that (:man_facepalming:  - unless I put the `(` in the search as well.... :roll_eyes: )